docs/contents/components/forms/dropzone/index.mdx

```diff
@@ -143,10 +145,24 @@ To specify the maximum number of uploadable files, set `maxFiles` to a number.
 ```tsx
 <Dropzone multiple maxFiles={3}>
   <VStack w="auto" gap="2xs">
-    <Text fontSize="xl">Drag file here or click to select file</Text>
-    <Text fontSize="sm">
-      Attach as many files as you like, can upload up to 3 files
-    </Text>
+    <For
```

Comment:
There is no need to use `For` here. I feel that the modification makes it less readable.

Reply:
Fixed.